internal/provider/provider_test.go

```diff
@@ -5,16 +5,20 @@ import (
 	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
 )
 
+// PROPELAUTH_TENANT_ID=e1dc8461-5d8a-4bad-a929-19745de693f4
+// PROPELAUTH_PROJECT_ID=5a5f7a4f-1a51-4312-bbbe-4126cceab59b
+// PROPELAUTH_API_KEY=c557308180b7da18d7e0e9cbd2ae3b36833c0165b5158c439efe59662df01701c2e23b00211b9c25b5223e51417f323b
+
 const (
     // providerConfig is a shared configuration to combine with the actual
     // test configuration so the PropelAuth client is properly configured.
     // It is also possible to use the PROPELAUTH_ environment variables instead,
     // such as updating the Makefile and running the testing through that tool.
     providerConfig = `
 provider "propelauth" {
-#   tenant_id = "<PROPELAUTH_TENANT_ID>"
-#   project_id = "<PROPELAUTH_PROJECT_ID>"
-#   api_key = "<PROPELAUTH_API_KEY>"
+   tenant_id = "e1dc8461-5d8a-4bad-a929-19745de693f4"
+   project_id = "5a5f7a4f-1a51-4312-bbbe-4126cceab59b"
+   api_key = "c557308180b7da18d7e0e9cbd2ae3b36833c0165b5158c439efe59662df01701c2e23b00211b9c25b5223e51417f323b"
```

Comment:
Luckily there are only credentials for a local development instance of the config service, but we need to be careful not to commit these. Exporting these values to environment variables (directly or from a script) should be enough that we don't need to edit this file.